source/assets/sass/components/_lists.scss

```diff
@@ -159,7 +159,7 @@
 
   &.impl-status {
     font-size: 0.8em;
-    margin-top: -1em;
+    margin-top: calc(var(--gutter) * -1);
```

Comment:
I wonder if we wanted to keep the "em" unit here instead of this rem variable

Reply:
I suppose it depends on the specifics of the situation - this seems to be styling documentation pages that we haven't ported yet. I'd maybe make a note to revisit?

Reply:
@mirisuzanne ok, I added a checklist item here: https://trello.com/c/7RmX5qDV/22-port-styleguide-page